templates/README.md

```diff
@@ -0,0 +1,116 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: http://34.68.228.168:8501/
```

Comment:
This tagger only has existing datasets.
When someone adds a dataset, it won't be listed in there yet (until it's merged on master).

Reply:
Indeed. @yjernite do you think you could share the code of the streamlit app so that people can run it on their own setup?

Reply:
I made a repo (private for now): https://github.com/huggingface/datasets-tagging

@lhoestq that's right. People can still use it to make yaml tags for any dataset without loading them in the menu for now, I'll look into loading a local dataset first thing Monday before I start working on my list.

Reply:
maybe by default it shouldn't even display a dataset when you open the Streamlit web app (blank dataset) but still let you write tags?

Reply:
Yup! I improved the app a bit, and it now loads an empty config. It can also load a local dataset once the CLI has created the `dataset_infos.json` file